bin/fake_puppet_db

```diff
@@ -6,4 +6,12 @@
 # end
 require_relative '../test/support/fake_puppet_db'
 
-Rack::Server.start(app: FakePuppetDB.new, Host: "localhost", Port: 8083)
+Rack::Server.start(
```

Comment:
Thats the important thing to run rails hdm (puma) and rails fake_puppet_db (webrick) at the same time.
Otherwise we had problems with colliding PIDs.

Reply:
@oneiros what do you think about that?

Reply:
It looks as if puma will always use the `config/puma.rb` configuration file if present. That is probably why the `pid` setting is overridden.

But if you look at that file you can see that this can actually be overridden by an environment variable.

So adding this line before `Rack::Server.start` did the trick for me:

```ruby
ENV["PIDFILE"] ||= "tmp/pids/fake_puppet_db.pid"
```